ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.17.yaml

```diff
@@ -319,25 +327,25 @@ tests:
       OCP_ARCH: arm64
       WORKER_INSTANCE_TYPE: m6g.xlarge
     workflow: openshift-e2e-aws-upi
-- as: ocp-e2e-aws-ovn-arm64-single-node
+- as: ocp-e2e-aws-ovn-arm64-sno-multi-a-a
```

Comment:
can we drop arm64 here?

Reply:
Good catch I missed this.  I'll update.

Reply:
Yep!  I looked right past this, my bad!  Will update.